src/main/java/org/opensrp/domain/UniqueId.java

```diff
@@ -56,7 +62,33 @@
 	@Column(name = COL_UPDATED_AT, columnDefinition = "TIMESTAMP DEFAULT CURRENT_TIMESTAMP")
 	@Temporal(TemporalType.TIMESTAMP)
 	private Date updatedAt;
+
+	@Column(name = COL_IDENTIFIER)
```

Comment:
This is not required since mybatis is being used instead of JPA or hibernate

Reply:
Done